Sources/UnidirectionalBinding.swift

```diff
@@ -312,15 +166,6 @@ public final class BindingTarget<Value>: BindingTargetProtocol {
 	public func consume(_ value: Value) {
 		setter(value)
 	}
-
-	@discardableResult
-	public static func <~ <Source: SignalProtocol>(target: BindingTarget<Value>, signal: Source) -> Disposable? where Source.Value == Value, Source.Error == NoError {
-		return signal
-			.take(during: target.lifetime)
-			.observeValues { [setter = target.setter] value in
-				setter(value)
```

Comment:
This should not be removed. It is intentionally override to capture `setter` directly, so that `self` need not be retained and/or uniqued by its parent. The default implementation weakly captures the target, so the binding would be invalid.

_(Seems like it wasn't covered in the spec.)_